mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
> You need the estimators for a lot of stuff (cross-condition generalization!!!, visualizing decoder topos, ...), 
+1

> but maybe there could be an option to dump them?

+1, PR welcome; it's an easy one.

> If the user wants all the estimators, they can simply do the loop and retain the ones they want.

If s/he does the loop him/herself, it'll take for ever. The advantage of the GAT is that is it fast, including for cross-condition generalization.